tests/test_xkrx_calendar.py

```diff
@@ -62,3 +62,36 @@ def test_constrain_construction_dates(self):
                 ' cannot instantiate the XKRX calendar for 2020.'
             )
         )
+
+    def test_holidays_fall_on_weekend(self):
+        # Holidays below falling on a weekend should
+        # not be made up during the week.
+        expected_holidays = [
+            # National Foundation Day on a Saturday
+            T('2015-10-03'),
```

Comment:
Since this is a Saturday (which is always not a session), I would actually instead test that the surrounding Friday and Monday are sessions (i.e. *not* holidays).